### PR TITLE
[KOGITO-6297] : User friendly deployment

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

https://issues.redhat.com/browse/KOGITO-6297
Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `main` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: process-usertasks-timer-quarkus-with-console/operator/kogito/process-usertasks-timer.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                name: kogito-infinispan-quarkus-config
+            - secretRef:
+                name: kogito-infinispan-truststore-quarkus-secret
+            - secretRef:
+                name: kogito-infinispan-quarkus-credential
+          volumeMounts:
+            - name: kogito-infinispan-truststore
+              mountPath: /home/kogito/certs/infinispan/truststore.p12
+              subPath: truststore.p12
+          image: 'quay.io/vajain/process-usertasks-timer-quarkus-with-console:30.0'
```
Comment (sutaakar) [inline process-usertasks-timer-quarkus-with-console/operator/kogito/process-usertasks-timer.yaml:57]: Would be better to have the image under kiegroup organization

**Review by sutaakar — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Review by spolti — COMMENTED** [review]
Sounds good, Vaibhav, can you please update the image name as Karel suggested?

**Comment by domhanak** [comment]
@ricardozanini should we revisit this PR?

**Comment by ricardozanini** [comment]
The Kogito Operator is not supported any longer.

**Pull request closed without merging**